src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectSystem.cs

```diff
@@ -207,7 +204,15 @@ private async Task LoadOrReloadProjectAsync(string projectPath, ProjectBuildMana
                 {
                     foreach (var logItem in loadedFile.Log)
                     {
-                        _logger.LogWarning($"{logItem.Kind} while loading {logItem.ProjectFilePath}: {logItem.Message}");
+                        if (logItem.Kind is WorkspaceDiagnosticKind.Failure)
+                        {
+                            _logger.LogError($"{logItem.Kind} while loading {logItem.ProjectFilePath}: {logItem.Message}");
+                        }
+                        else
+                        {
+                            _logger.LogWarning($"{logItem.Kind} while loading {logItem.ProjectFilePath}: {logItem.Message}");
+                        }
+                        await ProjectInitializationHandler.SendProjectLoadFailureNotificationAsync(logItem, cancellationToken);
```

Comment:
It may be good to also have a similar handler in the catch (Exception e) line at the end of this file.